lola.html

```diff
@@ -626,9 +648,16 @@ <h4>Keeping historical values</h4>
       </p>
 
       <p>
-        Most other values not specified here MAY change. A server may have a good reason to transform some values - for instance translating the ‘content’ value to a format that it supports, and/or translating the ‘type’ to a type it supports.  
+        Most other values not specified here MAY change. The 'content' or 'type' values MAY change.
```

Comment:
This might also justify a reference to the "best effort" philosophy, as well as an explicit punt to export/import as a better strategy if fine-grained fidelity or giving the user more info about (or even controls over) those transformations, retypings, dropped properties, etc.